### PR TITLE
 #150541483 Update contributions icons

### PR DESCRIPTION
**Pull request by Mbarak-Mbigo**
#### What does this PR do?
Update contributions icons

#### Description of Task to be completed?
Update contributions icons to match those in the mockup

#### How should this be manually tested?
After cloning the repo

- Run yarn install
- Run yarn start
- Access the landing page via localhost:3000
- Click on the LOGIN TO CONTRIBUTE button to log in

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[150541483](https://www.pivotaltracker.com/story/show/150541483)

#### Screenshots (if appropriate)
screen shot 2017-08-28 at 23 51 35

#### Questions:
N/A

### CODE REVIEW

File: src/pages/contributions/Contribution.scss [hunk 1]
```diff
@@ -155,6 +159,10 @@
```
Comment (andela-uofoegbu) [inline src/pages/contributions/Contribution.scss]: Get rid of extra new line

File: src/pages/contributions/ContributionsCard.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import { Icon } from 'react-inline-svg-icon';
 import UpvoteSection from '../../components/upvoteSection/upvoteSection.component';
 
 // styles
```
Comment (andela-uofoegbu) [inline src/pages/contributions/ContributionsCard.tsx]: If there's no imported stylesheet, then get rid of the comment

File: src/pages/contributions/Contribution.scss (hunk 1 above)
Comment (andela-uofoegbu) [inline src/pages/contributions/Contribution.scss]: Get rid of the extra line

Comment (Mbarak-Mbigo) [reply src/pages/contributions/Contribution.scss]: which extra line? I can't seem to find any that's unnecessary

Comment (andela-uofoegbu) [reply src/pages/contributions/Contribution.scss]: Okay. thought i saw one.

**Comment by BolajiOlajide** [comment]
LGTM

File: src/pages/contributions/Contribution.scss (hunk 1 above)
Comment (Mbarak-Mbigo) [reply src/pages/contributions/Contribution.scss]: 👍

**Pull request merged**